portfolio/src/main/webapp/index.html

```diff
@@ -81,9 +91,9 @@ <h2 class="title">About Me</h2>
         <div class="max-width">
             <div class="facts-content">
                 <div class="left">
-                    <p>Click here to get a random fun fact about The Office:</p>
-                    <button onclick="addRandomGreeting()">The Office Fact</button>
-                    <div id="greeting-container"></div>
+                    <p>Click here to get a random work/research experience of mine!</p>
+                    <button onclick="showContent()">Click Me!</button>
```

Comment:
nit: just a style difference, but I prefer adding event listeners over stringy-javascript-in-html. e.g.,.

```
let theButton = document.querySelectorAll('button#showContentButton')[0];
theButton.addEventListener('click', showContent);
```

Obviously, you'd need to add the id and double check I didn't do something silly here.